codegen/smithy-aws-swift-codegen/src/main/resources/META-INF/services/software.amazon.smithy.swift.codegen.integration.SwiftIntegration

```diff
@@ -25,7 +25,6 @@ software.amazon.smithy.aws.swift.codegen.model.AWSEndpointTraitTransformer
 software.amazon.smithy.aws.swift.codegen.model.AWSDeprecatedShapeRemover
 software.amazon.smithy.aws.swift.codegen.AWSClientConfigurationIntegration
 software.amazon.smithy.swift.codegen.swiftintegrations.InitialRequestIntegration
-software.amazon.smithy.aws.swift.codegen.swiftintegrations.RegistryConfigIntegration
```

Comment:
This integration used to write a `.swiftpm/configuration/registries.json` file for use with Swift Package Registry.  It is no longer used and is deleted.